README.md

```diff
@@ -2,6 +2,9 @@
 
 # Flexible Modeling System (FMS) Coupler
 
+Instructions for contribuing to this project can be found in the 
+[Contribution Guidelines](CONTRIBUTING.md)
```

Comment:
Might want to add a short blurb about this repo requiring a higher level of scrutiny as part of the need to read the Contributing Guide.